css/components/component.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="avatar.css">
+    <title>Component: Avatar</title>    
+    <style>
+        div{
```

Comment:
tag based styling is better to avoid and do not mix html and css unless it is highly necessary

Reply:
Hey @bhanuprakashj that file is for `preview-only` and I missed to remove before push.